### PR TITLE
Mat 8251 add purpose to exports

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE FHIR SERVICE

Jira Ticket: [MAT-8251](https://jira.cms.gov/browse/MAT-8251)
(Optional) Related Tickets:

### Summary
add purpose to exports

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/resources/templates/measure-metadata.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% if include.measure.identifier.exists() or include.measure.version.exists() or include.measure.scoring.exists() or
 include.measure.compositeScoring.exists() or include.measure.type.exists() or include.measure.riskAdjustment.exists() or
-include.measure.rateAggregation.exists() or include.measure.rationale.exists() or include.measure.clinicalRecommendationStatement.exists() or
+include.measure.rateAggregation.exists() or include.measure.rationale.exists() or include.measure.purpose.exists() or include.measure.clinicalRecommendationStatement.exists() or
```
Comment (adongare) [inline src/main/resources/templates/measure-metadata.html]: is this custom code or taken from liquid template repository?

Comment (sb-prateekkeerthi) [reply src/main/resources/templates/measure-metadata.html]: It is a custom code which I have added

**Review by adongare — APPROVED** [review]

File: src/main/java/gov/cms/madie/madiefhirservice/services/MeasureTranslatorService.java [hunk 2]
```diff
@@ -74,12 +74,12 @@ public org.hl7.fhir.r4.model.Measure createFhirMeasureForMadieMeasure(Measure ma
         .setCopyright(StringUtils.isBlank(copyright) ? UNKNOWN : copyright)
         .setDisclaimer(StringUtils.isBlank(disclaimer) ? UNKNOWN : disclaimer)
         .setRationale(rationale)
+        .setPurpose(madieMeasure.getMeasureMetaData().getPurpose())
         .setLibrary(
             Collections.singletonList(
                 new CanonicalType(
                     FhirResourceHelpers.buildResourceFullUrl(
                         "Library", madieMeasure.getCqlLibraryName()))))
-        .setPurpose(UNKNOWN)
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/madie/madiefhirservice/services/MeasureTranslatorService.java:82]: Is `UNKNOWN` an expected default value if the user hasn't populated `Purpose`?

Similar to how we handle `Copyright` and `Disclaimer`.

Comment (sb-prateekkeerthi) [reply src/main/java/gov/cms/madie/madiefhirservice/services/MeasureTranslatorService.java:82]: This field is optional, so if no information is set, the value should not be placed in the exports. --> this is what is mentioned in the ticket. so I am not displaying unknown (similar to rationale field when there is no value you don't show it at all)

**Review by jkotanchik-SB — APPROVED** [review]
